src/components/CreateGameDialog.vue

```diff
@@ -53,6 +53,7 @@
     <template #actions>
       <v-form>
         <v-btn
+          class="mr-2"
```

Comment:
@itsalaidbacklife this looks ok to me, pretty simple. You're more familiar with all the CSS stuff.

Reply:
Thanks.
Yeah, it's more similar to Tailwind.

Reply:
Yes this is right and yes it's a similar approach to tailwind. Vuetify bakes some of these styles in out of the box. Good for simple things imo